docs/src/user/basic_setup.md

```diff
@@ -20,7 +20,6 @@ To compile and run, Youki itself depends on some underlying libraries being inst
 $ sudo apt-get install    \
       pkg-config          \
       libsystemd-dev      \
-      libdbus-glib-1-dev  \
```

Comment:
😍

Reply:
@YJDoc2 May I ask you to write a migration guide to let the users know that `libdbus` can be deleted?

Reply:
Yes, I'll update the migration guide with changes in this commit.

Reply:
done